docs/installing-northstar/using-northstar/commands.md

```diff
@@ -48,6 +48,7 @@ bind "F5" "toggleconsole"
 | `ns_disallowed_tactical_replacement` | Name of optional replacement ability for disallowed tacticals                                               | ex: `"mp_ability_grapple"`                      |
 | `give mp_weapon_peacekraber`         | Gives you peacekraber                                                                                       |                                                 |
 | `r_latencyflex`                      | Enables [LatencyFleX](../../using-northstar/playing-on-linux/#latencyflex) (Linux-only, enabled by default) | `0` / `1`                                       |
+| `rui_drawEnable`                     | Enables or disables the HUD                                                                                 | `0` / `1`                                       |
```

Comment:
`rui_drawEnable` might even be in vanilla, need to double check with someone who knows ^^"

Reply:
We add it 

https://github.com/R2Northstar/NorthstarLauncher/blob/9b1e2a0f4895f48b6a8169bf1b571294b2ad4c54/NorthstarDLL/client/clientruihooks.cpp#L12

Reply:
Perfect, then it's already in the right section and can be merged as is ^^